<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WHT工厂工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabUdpDebug">
       <attribute name="title">
        <string>UDP调试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_udp">
        <item>
         <widget class="QGroupBox" name="groupBoxConfig">
          <property name="title">
           <string>UDP配置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelLocalIP">
             <property name="text">
              <string>本地IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditLocalIP">
             <property name="text">
              <string>192.168.0.3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelLocalPort">
             <property name="text">
              <string>本地端口:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditLocalPort">
             <property name="text">
              <string>8080</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRemoteIP">
             <property name="text">
              <string>目标IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRemoteIP">
             <property name="text">
              <string>192.168.0.2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRemotePort">
             <property name="text">
              <string>目标端口:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRemotePort">
             <property name="text">
              <string>8080</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonConnect">
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDisconnect">
             <property name="text">
              <string>断开</string>
             </property>
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxSend">
          <property name="title">
           <string>发送数据</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditSendData">
             <property name="placeholderText">
              <string>请输入十六进制数据，如: a5 cc ff</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSend">
             <property name="text">
              <string>发送</string>
             </property>
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClearSend">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxLog">
          <property name="title">
           <string>日志</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="textEditLog">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonClearLog">
               <property name="text">
                <string>清空日志</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDeviceManagement">
       <attribute name="title">
        <string>设备管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_device">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_device_query">
          <item>
           <widget class="QPushButton" name="pushButtonQueryDevices">
            <property name="text">
             <string>设备查询</string>
            </property>
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_device">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetDevices">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>设备ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>短ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>在线状态</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>版本</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>电池电量</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSlaveConfig">
       <attribute name="title">
        <string>从机配置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_slave_config">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_slave_config_actions">
          <item>
           <widget class="QPushButton" name="pushButtonAddSlaveConfig">
            <property name="text">
             <string>新建配置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_slave_config">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetSlaveConfigs">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>配置名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>配置信息</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>操作</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
